openweathermap-ios/openweathermap-iosTests/BookmarkLocation/Manager/BookmarkLocationManagerTest.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  BookmarkLocationServiceTest.swift
+//  openweathermap-ios
+//
+//  Created by Cassio Sousa on 24/04/20.
+//
+
+import XCTest
+
+class BookmarkLocationManagerTest: XCTestCase {
+
+    private var bookmarkLocationManager = BookmarkLocationManager()
+    
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    /**
+        It validates if not exits elementes
+     */
+    func testNoElementsBookmarkLocation() throws {
+        UserDefaults.standard.removeObject(forKey: BookmarLocationEnumaration.bookmarkLocation.rawValue)
```

Comment:
You are repeating too much `UserDefaults.standard` on your tests. Consider creating a property.

Reply:
Fixed in b6bb083